Tests/NineSunsSDKTests/AuthenticationTests/UserSignInTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Untitled.swift
```

Comment:
why is this Untitled?

Reply:
@masterfung Sorry, it was just a default comment when creating new file and I forgot to update it. Updated anyway.